### PR TITLE
Old PR's from repo and generifying docker-compose files

### PR DESCRIPTION
**Pull request by krisgeus**

### CODE REVIEW

File: chapter03/dags/03_with_end_date.py [hunk 1]
```diff
@@ -31,7 +31,10 @@ def _calculate_stats(input_path, output_path):
     calculate_stats = PythonOperator(
         task_id="calculate_stats",
         python_callable=_calculate_stats,
-        op_kwargs={"input_path": "/tmp/data/events.json", "output_path": "/tmp/data/stats.csv",},
+        op_kwargs={
+            "input_path": "/tmp/data/events.json",
```
Comment (krisgeus) [inline chapter03/dags/03_with_end_date.py:35]: /tmp is not needed. When using the new docker-compose /data is a volume in all containers.

File: chapter03/dags/02_daily_schedule.py [hunk 2]
```diff
@@ -30,7 +30,10 @@ def _calculate_stats(input_path, output_path):
     calculate_stats = PythonOperator(
         task_id="calculate_stats",
         python_callable=_calculate_stats,
-        op_kwargs={"input_path": "/tmp/data/events.json", "output_path": "/tmp/data/stats.csv",},
+        op_kwargs={
+            "input_path": "/tmp/data/events.json",
```
Comment (krisgeus) [inline chapter03/dags/02_daily_schedule.py:34]: /tmp is not needed see other comment

File: chapter03/dags/01_unscheduled.py [hunk 3]
```diff
@@ -27,7 +31,10 @@ def _calculate_stats(input_path, output_path):
     calculate_stats = PythonOperator(
         task_id="calculate_stats",
         python_callable=_calculate_stats,
-        op_kwargs={"input_path": "/tmp/data/events.json", "output_path": "/tmp/data/stats.csv",},
+        op_kwargs={
+            "input_path": "/tmp/data/events.json",
```
Comment (krisgeus) [inline chapter03/dags/01_unscheduled.py:35]: /tmp is not needed see other comment

File: chapter02/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,306 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (krisgeus) [inline chapter02/docker-compose.yml:1]: @ismaelcv For starting this airflow the chapter mentions using the shell script. Docker compose i use form chapter 3 on. I'm not against having the compose file here but it should be a deliberate choice.

File: chapter03/dags/03_with_end_date.py (hunk 1 above)
Comment (ismaelcv) [reply chapter03/dags/03_with_end_date.py:35]: Thats completely right, we already mounted the data directory as well, I will correct that

File: chapter02/docker-compose.yml (hunk 4 above)
Comment (ismaelcv) [reply chapter02/docker-compose.yml:1]: For development purposes i would like to include it. 
Its really handy to have a compose file there to be ready to run the examples

Comment (krisgeus) [reply chapter02/docker-compose.yml:1]: Fine with me.

**Comment by krisgeus** [comment]
@ismaelcv @jrderuiter Could we check this PR? I think at least the first chapters are fine now and we can do a chapter by chapter PR if we need to adjust.

**Review by ismaelcv — APPROVED** [review]

**Comment by ismaelcv** [comment]
@krisgeus I have approved. I also think is a good idea to merge what we have for now and take the next chapters on separate PRs

Thanks a lot for the review! really good insights specially in the DockerCompose setup.

**Pull request merged**